### DISCUSSION TITLE
Overriding default formatting

### DISCUSSION THREAD

**Discussion by timnevits** [chuangcaleb/obsidian-fountain-editor#31] [Q & A] [answered]
I'm a bit of a noob with both Obsidian and obsidian-fountain-editor, so hoping you can help me with the obvious. I see in the base.css that I can override the font with:

`--fountain--font-family:`

But where do I make this override? Do I edit the style.css file directly in:
`obsidian/plugins/fountain-editor`

Or, should I insert something into the yaml front matter?

If the latter, could you provide an example?  I'm trying the following with no luck:

```
---
cssclasses:
  - fountain
fountain--font-family:
  - 'Andale Mono'
---
```

**Comment by chuangcaleb** [comment, marked as answer]
Welcome to Obsidian! One thing that drew me in was being able to customize every element with [CSS snippets](https://help.obsidian.md/Extending+Obsidian/CSS+snippets). There are some great articles out there, let me know if you need more guidance, ya?

**Reply by timnevits** [reply]
Hey, thanks for tip!

On my iPad, for some reason css snippets didn't modify the fountain output on my iPad. I'll have to try on my Mac. BUT I was able to edit the style.css directly and get a thicker font :)

I'm learning... slowly. Thanks for all your work on this plugin!

**Reply by chuangcaleb** [reply]
I've not really gotten into Obsidian on mobile, I do all my things on desktop — let me know if you need anything else!
